docs/Data/Injector.md

```diff
@@ -0,0 +1,52 @@
+## Module Data.Injector
+
+#### `Injector`
+
+``` purescript
+data Injector a b
+  = Injector (a -> b) (b -> Maybe a)
```

Comment:
Fair enough... I don't think I'd advocate adding a dependency on the lens/optics stuff, unless it would provide significant value.

Another option would be to use profunctor prisms here, so that it would be type-compatible with other implementations. I suppose that would still require imposing a dependency on `purescript-profunctors`; anyway, I don't have a strong opinion about any of this, and I tend to bias toward local re-implementation rather than dependency bloat.

What do you think?